### DISCUSSION TITLE
Add Snatched Shows On Schedule Page

### DISCUSSION THREAD

**Discussion by HPMSGEN8** [pymedusa/Medusa#5160] [Ideas & Feature requests]
Hi Team

I am not sure if this is in the backlog or in the works but is there any chance to add the Snatched Shows on the Schedule Page ?

This way people can see what has been snatched and if anything was missed. 

Thanks

**Comment by sharkykh** [comment]
Related/Same: https://github.com/pymedusa/Medusa/issues/3170
It should be behind a flag that defaults to off.
